### ISSUE TITLE
Separate save paths for movies and shows

### ISSUE THREAD

**Issue by CndaRocx** [scakemyer/plugin.video.quasar#97]
I am loving what you have done so far, my family is happy to have this up and running again :) 

I use the "keep files after playback/download stops" feature, would it be possible to have the files automatically saved or moved to the same folders used for the library integration?  movies to movies folder, shows to shows folder. and/or an option to specify what content type goes were. I know this would make my life much easier.

**Comment by Magic815** [comment]
I think this would be a great thing to be implemented along with https://github.com/scakemyer/plugin.video.quasar/issues/14. See my comment in that issue on how it could be handled.

**Comment by scakemyer** [comment]
Very true, #14 is next after I fix a few bugs/regressions related to background downloads. Separate paths need to be thought out since the same path setting is used in lots of places so it's not that simple to make sure it will work well, and a little more work too ;)

**Comment by entr** [comment]
+1 looking forward this feature

**Comment by drake0101** [comment]
+1 also looking forward for this

**Comment by tomtomclub** [comment]
Yes  nice :)

**Comment by scakemyer** [comment]
So, I had a partly implemented WIP of this for a while, and got it mostly working yesterday, but it also confirmed a few concerns I had with this, namely that it becomes very difficult if not impossible to reliably play or resume torrents, since this creates a requirement to pass a content type parameter as well as a TMDB id every time we want to play something, instead of just a URI or existing torrent index. Other add-ons that use the `/play` or `/playuri` endpoints to add torrents to Quasar would also have to do this every time, otherwise we end up with files in both the subfolders and the main download folder also. You might then want to play the same torrent from Quasar, but it wouldn't find it in the Movies or Shows subfolder, so you'd go through the search/add process again and end up with duplicate files in both places. We could also check if it does exist in either places, but it gets ugly pretty fast having to do this.

Another issue has to do with the main reason we want to do it in the first place, namely to keep downloads in separate folders for the Kodi library. However, we'd be adding files in there while they're buffering, and eventually Kodi will scan those folders for new content, and hit a file that hasn't finished downloading, or worse, its initial buffering. I'm not sure if it would add it to its library anyway or not, but if it does, chances are it won't be able to get the proper metadata from the files, and we'd end up with a broken entry in the library which would surely require to be cleaned up manually. I haven't got the chance to test this, but I strongly suspect it would cause all sorts of problems down the line.

Bottom line is that I'm dropping this issue in favor of #440, which also takes over #536, as I think it will solve this in a much cleaner and more convenient way to keep our Kodi library well organized.
